fixtures/act/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Is there a reason this needs to be a separate fixture package, instead of adding it to the DOM fixtures one? It would be nice if we could start consolidating our fixtures into as few as possible.

Reply:
No real reason, I'll merge it with the dom fixture

Reply:
hrmm actually, I needed this specifically to set up a test environment differently from the dom fixture. the dom fixture uses built versions via public, while the act one needs it in node_modules. open to suggestions, but I just tried and it's pretty annoying. I expect this to also host some stuff I plan to do to test www style setups too. would like to keep it.

Reply:
Mainly I would like it to be the same package with the same package.json and node_modules. Ideally it would be a single app and single command that runs both, but even if they are separate commands that's better than nothing. Historically what happens is people will add these new fixtures and then nobody else opens them until somebody happens to notice that they are broken, by which time the original person has either lost context on what the desired behavior is, or they left the team entirely. The easier it is to run these fixtures, the more likely people will, I think.

Reply:
alright I updated the PR to merge the act/dom folders. someone still has to run yarn test in the dom folder, but at least it's all in one place

Reply:
this now runs in ci